src/OneBlinkFormBase.tsx

```diff
@@ -943,10 +943,14 @@ function OneBlinkFormBase({
         return
       }
 
-      setUnsavedChangesState((current) => ({
-        ...current,
-        isDirty: true,
-      }))
+      // Do not mark changes as unsaved when collapsing or expanding sections
+      if (element.type !== 'section') {
+        setUnsavedChangesState((current) => ({
+          ...current,
+          isDirty: true,
+        }))
```

Comment:
Should this go into the `else` block below? I think we want to apply the same logic as `section` to `calculation` and `summary`.

Reply:
Well it can't go in the `else` block, because thats for non sections. Its not in the AC, so I don't want to assume.

Reply:
Isn't that want me want? For it to only happen when the element that has changed is not a section.

Reply:
I like your reasoning ❤️